src/store/ducks/opensrp/jurisdictionsMetadata/index.ts

```diff
@@ -0,0 +1,86 @@
+import { AnyAction, Store } from 'redux';
```

Comment:
For such a simple reducer we should use [Reducer factory](https://github.com/OpenSRP/opensrp-web/tree/master/packages/opensrp-reducer-factory).  However, at this point I am fine of you create an issue for this to be done later.

Reply:
@KipSigei did you create the issue?

Reply:
@moshthepitt https://github.com/onaio/reveal-frontend/issues/1012